crates/core/executor/src/record.rs

```diff
@@ -383,6 +387,8 @@ impl MachineRecord for ExecutionRecord {
 
         self.memory_initialize_events.append(&mut other.memory_initialize_events);
         self.memory_finalize_events.append(&mut other.memory_finalize_events);
+
+        self.u256x2048_mul_events.append(&mut other.u256x2048_mul_events);
```

Comment:
Can you group this with the other precompile events above (e.g. around line 379).